devices/src/virtio/vsock/epoll_handler.rs

```diff
@@ -142,6 +213,10 @@ where
             }
         }
 
+        if raise_irq {
+            self.signal_used_queue().unwrap_or_default();
```

Comment:
Rust warns you when you're not handling Results in any way. This seems like a hack to silence the compiler about it. However, is there any good reason why this shouldn't return the error? At least logging such an event would be a better solution than simply failing silently.

Reply:
The error is already logged by `signal_used_queue()`.

Reply:
I was not referring to logging the error but to propagating it. Right now the `unwrap_or_default` is silencing it in case it returns error. 
Is that what you want? 
The net device is propagating it: https://github.com/firecracker-microvm/firecracker/blob/master/devices/src/virtio/net.rs#L581.